hoomd/md/export_PotentialPair.cc.inc

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2009-2022 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+#include "hoomd/md/PotentialPair.h"
+#include "hoomd/md/EvaluatorPair@_evaluator_cpp@.h"
+
+namespace hoomd { namespace md { namespace detail {
+
+void export_PotentialPair@_evaluator@(pybind11::module &m)
+    {
+    export_PotentialPair<EvaluatorPair@_evaluator_cpp@>(m, "PotentialPair@_evaluator@");
+    }
```

Comment:
This is not valid C++ syntax. The CMake command `configure_file` uses this as a template to generate valid C++ files at configure time: https://github.com/glotzerlab/hoomd-blue/blob/c87d24f91d8b70a40437dedb4d9710226880dcee/hoomd/md/CMakeLists.txt#L477-L523

The alternative is to create the 100+ files like this by hand rather than using loops in CMake with template files. They must be separate files so they can be compiled in parallel to build quickly and with low memory requirements.